### DISCUSSION TITLE
wgpu program: runtime error STATUS_STACK_BUFFER_OVERRUN

### DISCUSSION THREAD

**Discussion by meditating-monkey** [gfx-rs/wgpu#1544] [Q&A]
Whenever I try to run a program that uses wgpu(directly or through a library), the program exits itself and states that the error behind it is STATUS_STACK_BUFFER_OVERRUN. Is this problem only with my computer? If so, any ideas how to fix this?

**Comment by cwfitzgerald** [comment]
What version of wgpu are you using? It would help us the most if you could get us a backtrace of where it was crashing by running in a debugger.

**Reply by kvark** [reply]
Would also help if you run wgpu's own examples (from github, preferably).

**Reply by meditating-monkey** [reply]
Hi guys, the wgpu examples seem to run now properly. I just updated windows and tried it once again and it works just fine. Looks like the problem was specific to my OS version. Anyways, thank you all for your time 🙏🏻 😄

**Reply by kvark** [reply]
Interestingly, we discovered a strange behavior in Intel Vulkan driver, which allocates more than 256K on the stack. So applications may hit this if they are messing with thread stack sizes, like Gecko. See https://phabricator.services.mozilla.com/D118840

Maybe that's what you are seeing? A stack trace would help explaining this.
